0800-appendix-198215-dale.Rmd

```diff
@@ -121,22 +116,22 @@ hab_site %>% filter(site == my_site & location == 'us') %>% pull(comments)
 
 
 ## Stream Characteristics Upstream {.unnumbered}
 
 `r fpr_my_survey_dist(loc = 'us')` `r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-198215-03) - \\@ref(fig:photo-198215-06)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-198215-d02))."))`
-`r fpr_my_habitat_paragraph(loc = 'us')` There were numerous cascades that were approximately 1m in height at the time of survey, with deep outlet pools. The stream channel was frequently confined and restricted from lateral migration from the valley walls. Significant amounts of large woody debris were found in and around the stream, adding complexity. There were trace amount of gravels suitable for spawning. After the first 500m of the survey, there were logs placed within the streambed perpendicular to the flow direction at regular intervals. It is presumed these logs were placed to capture substrate and provide complexity and spawning/rearing habitat.  The stream was surveyed to the mapped location of a dam. Although the dam was not present there appeared to be some remnants of a historic structure in the form of logs `r if(gitbook_on){knitr::asis_output("(Figures \\@ref(fig:photo-198215-05)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-198215-d03))."))`  The habitat was rated as `r fpr::fpr_my_priority_info(loc = 'us')` value as an important migration corridor containing spawning habitat suitable for coho and steelhead and having moderate rearing potential for both species.
+`r fpr_my_habitat_paragraph(loc = 'us')` There were numerous cascades that were approximately 1m in height, with deep outlet pools. The stream channel was frequently confined and restricted from lateral migration from the valley walls. Significant amounts of large woody debris were found in and around the stream, adding complexity. There were trace amount of gravels suitable for spawning. After the first 500m of the survey, there were logs placed within the streambed perpendicular to the flow direction at regular intervals. It is presumed these logs were placed to capture substrate and provide complexity and spawning/rearing habitat.  The stream was surveyed to the mapped location of a dam. Although the dam was not present, there appeared to be some remnants of a historic structure in the form of logs `r if(gitbook_on){knitr::asis_output("(Figure \\@ref(fig:photo-198215-05)).")}else(knitr::asis_output("(Figure \\@ref(fig:photo-198215-d03))."))`  The habitat was rated as `r fpr::fpr_my_priority_info(loc = 'us')` value as an important migration corridor containing spawning habitat suitable for coho and steelhead and having moderate rearing potential for both species.
```

Comment:
>The stream was surveyed to the mapped location of a dam.

maybe from last author but there are two mapped locations of dams so we need to say which one this is referening too